app/server/appsmith-server/src/main/resources/application.properties

```
@@ -91,4 +91,4 @@ signup.allowed-domains=${APPSMITH_SIGNUP_ALLOWED_DOMAINS:}
 
 # Google recaptcha config
 google.recaptcha.key.site = ${APPSMITH_RECAPTCHA_SITE_KEY:}
-google.recaptcha.key.secret= ${APPSMITH_RECAPTCHA_SECRET_KEY:}
+google.recaptcha.key.secret= ${APPSMITH_RECAPTCHA_SECRET_KEY:}
```

There doesn't seem to be any change in this right? Just the blank line at the end of the file. Can we revert it?

```suggestion
google.recaptcha.key.secret= ${APPSMITH_RECAPTCHA_SECRET_KEY:}

```